third_party/ti_simplelink_sdk/cc13x2_26x2/crypto/aes_alt.c

```diff
@@ -0,0 +1,173 @@
+/******************************************************************************
```

Comment:
files like these for other targets are either placed in an SDK in a Docker image (e.g. efr32, espressif) or checked in as git submodules from a publicly available git repository (see e.g. qpg_sdk)  can you make that happen for these?

Reply:
didn't realize this would be a sticking point. I had assumed repackaging with a license and manifest would suffice, because that was what looked like was happening for lwip and jlink.

But I have gone through and removed the files originating from the TI SDK. Instead we are now building the files from one of the example applications within the TI SDK. This is a bit more coupled to the examples directory in the SDK than I would prefer, but it works. This also requires one manual change to the platform files when the SDK is installed due to a difference in OpenThread version.

Reply:
do you have levers to make the TI SDK materialize separately at some point?  that'd be cool if you did.  your solution is probably ok for now, not a lot of good choices without something installable from TI

Reply:
I just downloaded the SDK and I think putting these files in CHIP's open source repository might violate the SDK's license agreement.